nvdaHelper/vbufBackends/gecko_ia2/gecko_ia2.cpp

```diff
@@ -552,14 +558,13 @@ VBufStorage_fieldNode_t* GeckoVBufBackend_t::fillVBuf(IAccessible2* pacc,
 		|| role == ROLE_SYSTEM_BUTTONMENU
 		|| ((role == ROLE_SYSTEM_CHECKBUTTON || role == ROLE_SYSTEM_RADIOBUTTON) && !isLabelVisible(pacc));

```

Comment:
`isLabelVisible()` is called here and on line 563. I wonder if this is intentional, should the bool `labelVisible` which is created at line 561 be used instead?